engine/platform/posix/sys_posix.c

```diff
@@ -158,9 +158,11 @@ void Platform_Shutdown( void ) {}
 double Platform_DoubleTime( void )
 {
 	struct timespec ts;
-
+	#if XASH_IRIX
```

Comment:
Unneeded indentation for #if/#else/#endif.

Also, please add the space after opening and before closing parentheses everywhere. Consecutive parentheses must not have spaces in between.

While style rules wasn't followed everywhere, it's better good to slowly put in order such lines.